packages/client/src/vscode-compatibility.ts

```diff
@@ -5,4 +5,6 @@
 import { createVSCodeApi } from "./vscode-api";
 import { Services } from "./services";
 
-export = createVSCodeApi(Services.get);
\ No newline at end of file
+const vscodeCompatibility = createVSCodeApi(Services.get);
+
+export { vscodeCompatibility };
```

Comment:
I'm surprised it work, the previous syntax was exporting each field of the api as a different named export, now you just export a `vscodeCompatibility`